arbitrum-docs/build-decentralized-apps/how-to-use-timeboost.mdx

```diff
@@ -0,0 +1,473 @@
+---
+title: 'How to use Timeboost'
+description: Learn how to use timeboost
+author: jose-franco
+content_type: how-to
+---
+
+Timeboost is a new transaction ordering policy for Arbitrum chains. With Timeboost, anyone can bid for the right to access an express lane on the sequencer for faster transaction inclusion.
+
+In this how-to, you'll learn how to bid for the right to use the express lane, how to submit transactions through the express lane, and how to transfer that express lane rights to someone else. To learn more about Timeboost, refer to the [gentle introduction](/how-arbitrum-works/timeboost/gentle-introduction.mdx).
+
+This how-to assumes that you're familiar with:
+
+- [How Timeboost works](/how-arbitrum-works/timeboost/gentle-introduction.mdx)
+- [viem](https://viem.sh/), since the snippets of code present in the how-to use this library
+
+## How to submit bids for the right to be the express lane controller
+
+To use the express lane for faster transaction inclusion, you must win an auction for the right to be the express lane controller for a specific round.
+
+::::info
+
+Remember that, by default, each round lasts 60 seconds, and the auction for a specific round closes 15 seconds before the round starts. These default values can be configured on a chain using the `roundDurationSeconds` and `auctionClosingSeconds` parameters, respectively.
+
+::::
+
+Auctions are held in an auction contract, and bids are submitted to an autonomous auctioneer, that also communicates with the contract. Let's take a look at the process of submitting bids and finding out the winner of an auction.
+
+### Step 0: gather required information
+
+Before we begin, make sure you have:
+
+- Address of the auction contract
+- Endpoint of the autonomous auctioneer
+
+### Step 1: deposit funds into the auction contract
+
+Before bidding on an auction, we need to deposit funds in the auction contract. These funds are deposited in the form of the ERC-20 token used to bid, also known as, the `bidding token`. We will be able to bid for an amount that is equal to or less than the tokens we have deposited in the auction contract.
```

Comment:
```suggestion
Before bidding on an auction, we need to deposit funds in the auction contract. These deposited funds are the ERC-20 token used to bid, also known as the `bidding token`. We will be able to bid for an amount that is equal to or less than the tokens we have deposited in the auction contract.
```

Reply:
I'd like to be a bit more explicit with what I'm referring to with "funds", since it can mean different things. I've kept the sentence as it was, but happy to change it to something that keeps my intention